umbraco/Umbraco-CMS
# Fixed profiling status endpoints and added an update endpoint

Fixed profiling endpoint and added a PUT to update the value.

Also cleaned up the usage of the cookie, so all access to that cookie is by using the service.

Test:
- Verify the endpoints work
- Verify the miniprofiler is activated when enabled and debug is configured in appsettings
  -  [/umbraco/profiler/results-index](https://localhost:44339/umbraco/profiler/results-index)
